code/game/gamemodes/cult/ghosts.dm

```diff
@@ -0,0 +1,236 @@
+
+	var/list/obj/item/choices = list()
+	for(var/obj/item/I in range(1))
+		if(I.w_class <= 2)
+			choices += I
+
+	if(!choices.len)
+		src << "<span class='warning'>There are no suitable items nearby.</span>"
+		return
+
+	var/obj/item/choice = input(src, "What item would you like to pull?") in null|choices
+	if(!choice || choice.w_class > 2)
+		return
+
+	if(step_to(choice, T))
+		choice.visible_message("<span class='warning'>\The [choice] suddenly moves!</span>")
+
+	ghost_magic_cd = world.time + 60 SECONDS
+
+/mob/dead/observer/proc/whisper_to_cultist(var/anyone = 0)
+	set category = "Cult"
+	set name = "Whisper to mind"
+	set desc = "Whisper to a human of your choice. If they are adjusted enough, they'll hear you."
+
+	if(ghost_magic_cd > world.time)
+		src << "<span class='notice'>You need some more time before you can use your abilities.</span>"
+		return
+
+	var/list/mob/living/choices = list()
+	for(var/mob/living/M in range(1))
+		choices += M
+
+	var/mob/living/choice = input(src, "Whom do you want to whisper to?") in null|choices
+	if(!choice)
+		return
+
+	var/message = sanitize(input("Decide what you want to whisper.", "Whisper", ""))
+
+	if(message)
+		if(iscultist(choice) || anyone)
+			choice << "<span class='notice'>You hear a faint whisper... It says... \"[message]\"</span>"
+		else
+			choice << "<span class='notice'>You hear a faint whisper, but you can't make out the words.</span>"
+		src << "You whisper to \the [choice]. Perhaps they heard you."
+
+	ghost_magic_cd = world.time + 100 SECONDS
+
+/mob/dead/observer/proc/bite_someone()
+	set category = "Cult"
+	set name = "Bite"
+	set desc = "Bite or scratch someone."
+
+	if(ghost_magic_cd > world.time)
+		src << "<span class='notice'>You need some more time before you can use your abilities.</span>"
+		return
+
+	var/list/mob/living/carbon/human/choices = list()
+	for(var/mob/living/carbon/human/H in range(1))
+		choices += H
+
+	var/mob/living/carbon/human/choice = input(src, "Whom do you want to scratch?") in null|choices
+	if(!choice)
+		return
+
+	var/method = pick("bit", "scratched")
+	choice << "<span class='danger'>Something invisible [method] you!</span>"
+	choice.apply_effect(5, AGONY, 0)
+	src << "<span class='notice'>You [method] \the [choice].</span>"
+
+	ghost_magic_cd = world.time + 60 SECONDS
+
+/mob/dead/observer/proc/chill_someone()
+	set category = "Cult"
+	set name = "Chill"
+	set desc = "Pass through someone, making them feel the chill of afterlife for a moment."
+
+	if(ghost_magic_cd > world.time)
+		src << "<span class='notice'>You need some more time before you can use your abilities.</span>"
+		return
+
+	var/list/mob/living/carbon/human/choices = list()
+	for(var/mob/living/carbon/human/H in range(1))
+		choices += H
+
+	var/mob/living/carbon/human/choice = input(src, "Whom do you want to scare?") in null|choices
+	if(!choice)
+		return
+
+	choice << "<span class='danger'>You feel as if something cold passed through you!</span>"
```

Comment:
After some thinking, I've decided to not make snowflake checks and leave it as it is on "It's magic, I don't have to explain it" explanation.

Plus, they can overheat IIRC, so it makes sense that they can feel their temperature.